tests/load/README.md

```diff
@@ -94,23 +94,13 @@ make load-clean
 
 Follow the steps bellow to execute the distributed load tests on GCP:
 
-### Calibration
-
-Following the addition of new features, such as a Locust Task or Locust User, it may be necessary
-to re-establish the recommended parameters of a performance test.
-
-| Parameter         | Description                                                                                                                                                                                                      |
-|-------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| [WAIT TIME][13]   | - Changing this cadence will increase or decrease the number of channel subscriptions and notifications sent by an AutopushUser. <br/>- This value can be set in the Locust UI.                                  |
-| [TASK WEIGHT][14] | - Changing this weight impacts the probability of a task being chosen for execution. <br/>- This value is hardcoded in the task decorators of the AutopushUser class.                                            |
-| USERS PER WORKER  | - This value should be set to the maximum number of users a Locust worker can support given CPU and memory constraints. <br/>- This value is hardcoded in the AutopushLoadTestShape class.                       |
-| LOCUST WORKERS    | - This value is derived by dividing the total number of users needed for the performance test by the 'USERS PER WORKER'. <br>- This value is hardcoded in the AutopushLoadTestShape and the setup_k8s.sh script. |
-
 ### Setup Environment
 
 #### 1. Start a GCP Cloud Shell
 
 The load tests can be executed from the [contextual-services-test-eng cloud shell][15].
+If executing a load test for the first time, the git autopush repository will need to
+be cloned locally.
```

Comment:
Maybe a note that someone should use the HTTPS git clone command? Since SSH isn't enabled in that context.

Reply:
```suggestion
be cloned locally into the root directory of the machine. Use the HTTPS git clone functionality as SSH is not supported in this environment.
```